### PR TITLE
doc(azure-iot-device): add new PnP samples

### PR DESCRIPTION
**Pull request by vishnureddy17**
Still need to update the README in the pnp samples folder.

### CODE REVIEW

File: device/samples/pnp/temperature_controller.js [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+const deviceSecurityType = process.env.IOTHUB_DEVICE_SECURITY_TYPE;
+
+// Must be set if IOTHUB_DEVICE_SECURITY_TYPE environment variable is "connectionString"
+let deviceConnectionString = process.env.IOTHUB_DEVICE_CONNECTION_STRING;
+
+// Must be set if IOTHUB_DEVICE_SECURITY_TYPE environment variable is "DPS"
+const provisioningHost = process.env.IOTHUB_DEVICE_DPS_ENDPOINT || 'global.azure-devices-provisioning.net';
+const idScope = process.env.IOTHUB_DEVICE_DPS_ID_SCOPE;
+const registrationId = process.env.IOTHUB_DEVICE_DPS_REGISTRATION_ID;
+const symmetricKey = process.env.IOTHUB_DEVICE_DPS_REGISTRATION_KEY;
```
Comment (vishnureddy17) [inline device/samples/pnp/temperature_controller.js:24]: Before, these environment variables were `IOTHUB_DEVICE_DPS_DEVICE_ID` and `IOTHUB_DEVICE_DPS_DEVICE_KEY`. This naming seems incorrect to me, so I changed it. However, I think the other languages use the old name. Not sure what we should be using here.

Comment (anthonyvercolano) [reply device/samples/pnp/temperature_controller.js:24]: As I recall, program managers spent a lot of time working on these names.  Do you REALLY want to be in disagreement with the other SDKs on this?

**Review by anthonyvercolano — APPROVED** [review]

File: device/samples/pnp/temperature_controller.js (hunk 1 above)
Comment (vishnureddy17) [reply device/samples/pnp/temperature_controller.js]: Changed it back.

**Pull request merged**